docs/detections/session-view.asciidoc

```diff
@@ -84,4 +93,37 @@ Session View includes an additional badge not pictured above:
 * The *Exec user change* badge highlights exec user changes, such as when a user escalates to root:
 +
 [role="screenshot"]
-image::images/session-view-exec-user-change-badge.png[The Exec user change button,width=80%,height=80%]
+image::images/session-view-exec-user-change-badge.png[The Exec user change badge,width=80%,height=80%]
+
+* The *Output* badge appears next to commands which generated TTY output. Click it to open Terminal output view.
++
+[role="screenshot"]
+image::images/session-view-output-badge.png[The Output badge,width=80%,height=80%]
+
+[[session-view-output]]
+[discrete]
+=== Terminal output view UI
+
+beta::[]
+
+Terminal output view helps provide a more complete understanding of commands executed by users or adversaries, and assists with auditing and compliance objectives. It allows you to review a session's TTY output, in the context of the commands which generated it. It includes both TTY output, and shell commands (if the shell was in echo mode).
```

Comment:
Editing out some filler words and comma splices. Don't know if "original commands" is the right phrase, but "commands which generated it" feels too wordy.

Also, i'm not sure about the last part "as typed -- provided the shell etc": The way it's phrased, it makes me wonder, well what if it _wasn't_ in echo mode? Wouldn't most bad actors turn off echo to hide their actions? Also "as typed" implies that every command is being manually typed by a human sitting at a keyboard, but I assume most malicious commands are scripted, API'd, or somehow automated. I might not understand the context of this feature, though -- maybe related to needing to explain what "TTY" means, and what exactly this output represents?
```suggestion
Terminal output view provides a more complete understanding of commands executed by users or adversaries, and assists with auditing and compliance. It allows you to review a session's TTY output in the context of the original commands. It includes both TTY output and shell commands (if the shell was in echo mode).
```